plugins/destination/azblob/client/spec.go

```diff
@@ -0,0 +1,39 @@
+package client
+
+import (
+	"fmt"
+)
+
+type FormatType string
+
+const (
+	FormatTypeCSV  = "csv"
+	FormatTypeJSON = "json"
+)
+
+type Spec struct {
+	Bucket   string     `json:"bucket,omitempty"`
+	Container string `json:"container,omitempty"`
+	Path     string     `json:"key,omitempty"`
```

Comment:
The field is called `Path` but the json is `key` - not sure which one we prefer but let's be consistent either way

Reply:
I changed it to `Path` in all places. good catch on the `json` tag